cmd/eksctl-anywhere/cmd/generateclusterconfig.go

```diff
@@ -156,6 +156,7 @@ func generateClusterConfig(clusterName string) error {
 			v1alpha1.ControlPlaneConfigCount(2),
 			v1alpha1.ExternalETCDConfigCount(3),
 			v1alpha1.WorkerNodeConfigCount(2),
+			v1alpha1.WorkerNodeConfigName(constants.DefaultWorkerNodeGroupName),
```

Comment:
I would expect all commands to remain untouched in a PR that only adds e2e tests
just for my understanding, why was this needed?
Also, aren't worker node config names set by default already?

Reply:
Without this added line, a worker node group will not have name, and there is no way to add a label to a worker node group without name. 

I do notice that there is default logic to set a `md-0` if no name is specified, and it is invoked when running `create/upgrade cluster`. But that default logic is not invoked before e2e trying to set up the config (adding label). 

If you check the same file of other providers (docker/vsphere/snow/tinkerbell/nutanix, they all have a line like #159